### ISSUE TITLE
Empty message box during firmware update

### ISSUE THREAD

**Issue by brevilo** [Nitrokey/nitrokey-app2#315]
Version: 2.3.3
OS: macOS 14.7.1

When updating the firmware the device reboots into bootloader mode. At that point a second message box appears. Apart from an exclamation mark, `Cancel` and `OK` (default) buttons, **no message is shown**. Hitting `OK` continues the update but I'd really like to make an informed decision :) I could reproduce this with a second device.

Also, after a first firmware update any subsequent update (same or other device) doesn't show the please-confirm-by-touching (at the start of the process) message anymore.

Cheers

**Comment by robin-nitrokey** [comment]
Thanks for the report!  The empty message box is a known problem in v2.3.3, see the [release notes](https://github.com/Nitrokey/nitrokey-app2/releases/tag/v2.3.3), and can be ignored safely.

**Comment by brevilo** [comment]
Yep, I figured that's the one. Feel free to close the issue.

**Comment by robin-nitrokey** [comment]
I think it’s not fixed yet so I’d keep it open for the moment.

**Comment by daringer** [comment]
thx for the report @brevilo - you are on Mac, could you share how you installed the nitrokey-app2 ?

**Comment by brevilo** [comment]
PyPI (#289)

**Comment by daringer** [comment]
oh nice, thx forgot about this issue - can you elaborate how exactly ? pipx? maybe even somehing like a ELI5 for other mac users? I'd like to add this [here](https://docs.nitrokey.com/software/nk-app2/#) as a usable installation way for mac.

**Comment by brevilo** [comment]
Always [Pipenv](https://pipenv.pypa.io/en/latest/) 😎
